samples/HostingPlayground/HostingPlayground.csproj

```diff
@@ -7,17 +7,9 @@
     <LangVersion>8</LangVersion>
   </PropertyGroup>
 
-  
   <ItemGroup>
     <ProjectReference Include="..\..\src\System.CommandLine.Hosting\System.CommandLine.Hosting.csproj" />
     <ProjectReference Include="..\..\src\System.CommandLine\System.CommandLine.csproj" />
   </ItemGroup>
 
-  
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.4" />
-    <PackageReference Include="microsoft.extensions.hosting" Version="3.1.4" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.4" />
-  </ItemGroup>
-
 </Project>
```

Comment:
In .NET Standard 2.1, the `Microsoft.Extensions.Hosting` package includes all of the necessary Generic Host infratrstructure (much like the ASP.NET Core `App` and `All` metapackages did in ASP.NET Core land). It is therefore no longer necessary to refer to single Generic Host packages individually. Doing so might actually cause version conflicts.